tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -246,8 +268,12 @@ async function connectCommerceMock(mockHost, tokenData) {
   }
 }
 
-async function ensureCommerceMockWithCompassTestFixture(client, appName, scenarioName, mockNamespace, targetNamespace) {
-  const mockHost = await provisionCommerceMockResources(`mp-${appName}`, mockNamespace, targetNamespace);
+async function ensureCommerceMockWithCompassTestFixture(client, appName, scenarioName, mockNamespace, targetNamespace, withCentralApplicationGateway=false) {
+  const mockHost = await provisionCommerceMockResources(
+      `mp-${appName}`,
+      mockNamespace,
+      targetNamespace,
+      withCentralApplicationGateway ? prepareLastorderCentralApplicationGatewayAndCompassObjs(`mp-${appName}`) : lastorderObjs);
```

Comment:
This will be simplified, since the `lastorderObjs` would have been already prepared and no adjustments at this place would be necessary.

Reply:
The function resource contains a token for replacement. Is this solution ok for you? The token is being replaced with different URL for different scenarios

Reply:
OK